walkthroughs/howto-tls-file-provided/README.md

```diff
@@ -293,7 +263,13 @@ Now when you call the service, you will see both `white` and `green` again.
 curl "${COLORAPP_ENDPOINT}/color"
 ```
 
-### Step 10: Clean Up
+Tip: You can use this command to clear the color teller history
```

Comment:
This threw me off until I read forward and saw the clear command :P

Reply:
Yeah I agree. We can change the service to return an error as a separate response and not in the weighted response, but for now I'm going to leave it.